targets/targets.json

```diff
@@ -3630,12 +3630,19 @@
             }
         },
         "overrides": {
-            "lse_available": 0
+            "lse_available": 0,
+            "boot-stack-size": "0x400",
+            "tickless-from-us-ticker": true
```

Comment:
I think this is not needed as TICKLESS is not enabled ?

Reply:
It is needed to avoid hitting https://github.com/ARMmbed/mbed-os/blob/master/platform/source/mbed_os_timer.cpp#L55

Reply:
I removed my last comment as discussion has already been started on this topic.

@kjbracey-arm 
init_os_timer function looks to me strange...
If target doesn't support LPTICKER, and then not TICKLESS... why we have to set "tickless-from-us-ticker" ?
Thx